packages/components/src/List/ItemTitle/ItemTitle.component.js

```diff
@@ -14,12 +14,23 @@ const ENTER_KEY = 13;
 
 export function RenderButton({ id, value, className, item, onClick }) {
 	function click(event) {
-		event.stopPropagation();
-		onClick(event, item);
+		// only fires onclick when left-click
+		if (event.button === 0) {
```

Comment:
In TDS-side, in callback `onClick` we do a redirection on a new page (`target = _blank`)
In that case, Chrome and FF behave differently when we do a middle-click (`mouseDown`) event . Chrome will not trigger a mouse click event after `mouseDown` event, but FF will. 
With this test, I prevent FF (and surely other browsers) to trigger a `onClick` event when there was a `mouseDown` event triggered before.